advanced/magicmethods.md

```diff
@@ -155,8 +152,7 @@ follow one of these styles:
     ...         self.name = name
     ...         self.founding_year = founding_year
     ...     def __repr__(self):
-    ...         return 'Website(name=%r, founding_year=%r)' % (
-    ...             self.name, self.founding_year)
+    ...         return f'Website(name={self.name}, founding_year={self.founding_year})'
```

Comment:
These too need repr. You want it to show `Website(name='github.com', ...)` instead of the more confusing `Website(name=github.com, ...)`

```suggestion
    ...         return f'Website(name={repr(self.name)}, founding_year={repr(self.founding_year)})'
```